build.gradle.kts

```diff
@@ -168,7 +173,16 @@ release {
     tagTemplate = "\$version"
 }
 
-tasks.afterReleaseBuild {
-    // bintray is being sunsetted
-    //dependsOn(tasks.named("bintrayUpload"))
+nexusPublishing {
+    repositories {
+        sonatype {
+            // sonatypeUsername and sonatypePassword properties are used automatically
+            // id found via clicking the desired profile in the web ui and noting the url fragment
+            stagingProfileId.set("TODO")
```

Comment:
@lemire will need the ID from you for this

Reply:
I do have a user account on oss.sonatype.org with user ID `lemire`.  Is that what goes there?

Reply:
That would be too easy. If you go to https://oss.sonatype.org/#stagingProfiles and select the profile you want, the url will change to `https://oss.sonatype.org/#stagingProfiles;ab8c5618978d18` (for my `org.mpierce` profile). The `ab8c5618978d18` is the part we need. This took a while for me to figure out...

Reply:
So maybe 3a809bf4297e26

See screenshot.

<img width="751" alt="Screen Shot 2021-04-15 at 9 22 45 AM" src="https://user-images.githubusercontent.com/391987/114876341-3a2f4e00-9dcc-11eb-8932-4f50b1dc99e9.png">

Reply:
That looks like the right thing. Can you give this branch a try with:

`./gradlew -Pversion=0.0 publishToSonatype`

That should create a staging repo and populate it with a fake version's worth of artifacts. You can then try "closing" the repo to see what errors pop up, if any. Either way, you can then "drop" the repo to discard it.

You'll need the following in your `~/.gradle/gradle.properties`:

```
sonatypeUsername = ...
sonatypePassword = ...

signing.keyId = ...
signing.password = ...
signing.secretKeyRingFile = ...
```

The [Gradle signing plugin](https://docs.gradle.org/current/userguide/signing_plugin.html) has some info on how to set up keys. I know almost nothing about GPG (other than that it is widely scorned by cryptographers), and I was able to muddle through it. Hopefully you already have some signing keys set up.

Reply:
Let me try.